### PR TITLE
add default metrics test

### PR DESCRIPTION
**Pull request by songleo**
fix https://github.com/open-cluster-management/backlog/issues/17909

### CODE REVIEW

File: tests/ignored-metric-list [hunk 1]
```diff
@@ -0,0 +1,79 @@
+authenticated_user_requests
```
Comment (clyang82) [inline tests/ignored-metric-list:1]: You may create a sub-folder `tests/pkg/testdata` to store this ignore-metrics-list.

File: tests/pkg/utils/client.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ func GetKubeClientDynamic(opt TestOptions, isHub bool) dynamic.Interface {
 	if !isHub && len(opt.ManagedClusters) > 0 {
 		url = opt.ManagedClusters[0].ClusterServerURL
 		kubeConfig = opt.ManagedClusters[0].KubeConfig
-		kubeContext = ""
+		kubeContext = opt.ManagedClusters[0].KubeContext
```
Comment (clyang82) [inline tests/pkg/utils/client.go:34]: It should be addressed in your previous PR. correct?

Comment (songleo) [reply tests/pkg/utils/client.go:34]: yes, let me rebase it, thanks.

**Pull request merged**